components/staking/TableValidators.vue

```diff
@@ -1,33 +1,23 @@
 <template>
-  <div class="container">
-    <table v-if="showingValidators.length" class="data-table">
-      <thead>
-        <PanelSort
-          :sort="sort"
-          :properties="properties"
-          :show-on-mobile="showOnMobile"
-        />
-      </thead>
-      <tbody
-        v-infinite-scroll="loadMore"
-        infinite-scroll-distance="400"
-        name="flip-list"
-      >
-        <LiValidator
-          v-for="(validator, index) in showingValidators"
-          :key="validator.operatorAddress"
-          :index="index"
-          :validator="validator"
-          :delegation="getDelegation(validator)"
-          :rewards="getRewards(validator)"
-          :show-on-mobile="showOnMobile"
-          :staking-denom="stakingDenom"
-        />
-      </tbody>
-    </table>
-    <div v-else-if="!searchTerm" class="loading-row">Loading...</div>
-    <div v-else class="no-results">No results</div>
-  </div>
+  <TableContainer
+    :length="showingValidators.length"
+    :columns="properties"
+    :sort="sort"
+    :loaded="loaded"
```

Comment:
Not so easy to fix the broken loading logic. I pass `loadMore` as a prop but doesn't seem to be triggering. Probably not updating the `showing` in the parent component

Reply:
OK, need to use the emit thing